--- ecore-1.2.0/m4/ecore_check_options.m4.orig	2012-02-20 19:57:33.000000000 +0100
+++ ecore-1.2.0/m4/ecore_check_options.m4	2012-05-28 19:49:08.231684650 +0200
@@ -178,8 +178,8 @@
    if test "x$_ecore_have_gnutls" = "xyes";then
      AC_PATH_GENERIC([libgcrypt], [], [_ecore_have_gnutls="yes"], [_ecore_have_gnutls="no"])
         if test "x${_ecore_have_gnutls}" = "xyes" ; then
-           TLS_CFLAGS+=" ${LIBGCRYPT_CFLAGS}"
-           TLS_LIBS+=" ${LIBGCRYPT_LIBS}"
+           TLS_CFLAGS="$TLS_CFLAGS ${LIBGCRYPT_CFLAGS}"
+           TLS_LIBS="$TLS_LIBS ${LIBGCRYPT_LIBS}"
         fi
    fi
 
